packages/axios-extension/src/auth/uaa.ts

```diff
@@ -179,24 +178,27 @@ export class Uaa {
      */
     protected async getAuthCode(timeout: number = defaultTimeout): Promise<{ authCode: string; redirect: Redirect }> {
         return new Promise((resolve, reject) => {
-            const app = express();
-            const server = http.createServer(app);
             // eslint-disable-next-line prefer-const
             let redirect: Redirect;
+            // eslint-disable-next-line prefer-const
+            let server: http.Server;
             const handleTimeout = (): void => {
-                server.close();
+                server?.close();
                 reject(new UAATimeoutError(`Timeout. Did not get a response within ${prettyPrintTimeInMs(timeout)}`));
             };
             const timer = setTimeout(handleTimeout, timeout);
-            app.get(Redirect.path, (req, res) => {
-                res.set('Content-Type', 'text/html');
-                res.send(Buffer.from(redirectSuccessHtml(this.logoutUrl, this.systemId)));
-                this.log.info('Got authCode');
-                resolve({ authCode: req.query.code + '', redirect });
-                if (timer) {
-                    clearTimeout(timer);
+            server = http.createServer((req, res) => {
+                const reqUrl = new URL(req.url, `http:${req.headers.host}`);
```

Comment:
Question about the interpolated base URL. Same as: https://github.com/SAP/open-ux-tools/pull/540/files#r974651046

Reply:
https://github.com/SAP/open-ux-tools/pull/540/commits/6eca7ee931b046459cc6f27a29d3d4ea4d99d050